Makefile

```
@@ -0,0 +1,12 @@
+.PHONY: all $(shell ls -d */ | sed 's/\///g')
+
+all: $(shell ls -d */ | sed 's/\///g' | grep -v '^\.')
```

We could list directories to have more control over the Makefile.

```suggestion
PROJECTS += entrypoint-0.6
PROJECTS += entrypoint-0.7
PROJECTS += entrypoint-0.8

.PHONY: all $(PROJECTS)

all: $(PROJECTS)
```

---

I was trying to minimize the list of changes on each commit of a new project